examples/movie_view_ratings/run_on_beam.py

```diff
@@ -60,21 +62,25 @@ def main(unused_argv):
 
         explain_computation_report = pipeline_dp.ExplainComputationReport()
         # Calculate the private sum
+        params = SumParams(
+            # Limits to how much one user can contribute:
+            # .. at most two movies rated per user
+            max_partitions_contributed=2,
+            # .. at most one rating for each movie
+            max_contributions_per_partition=1,
+            # .. with minimal rating of "1"
+            min_value=1,
+            # .. and maximum rating of "5"
+            max_value=5,
+            # The aggregation key: we're grouping data by movies
+            partition_extractor=lambda mv: mv.movie_id,
+            # The value we're aggregating: we're summing up ratings
+            value_extractor=lambda mv: mv.rating)
+        if FLAGS.pre_threshold:
+            params.pre_threshold = FLAGS.pre_threshold
```

Comment:
I would have a preference for defining it at the same level as the other parameters rather than as a flag, because it's not very clear why we specify parameters in different places.